facia-tool/public/css/style.css

```diff
@@ -1523,6 +1523,9 @@ button {
     text-align: center;
 }
 
+.tool--small--copy,
```

Comment:
This is smelly. For me creating modifications of modifications is a bad practice.

Look at it this way:

You have 'tool' object which has some default tool styling.
You have modificator 'tool--small' which makes tool object smaller.

And now you have 'tool--small--paste' which adds some additional style to 'tool--small'. What if you will need add those styles to 'tool--large' in the future? Will you copy styles to a new class named 'tool--large--paste'?

It makes sense to create another modification 'tool--paste' which will add only 'paste' styles without knowing if the element is small, big or whatever size.

So you will end up with class="tool tool--small tool--paste" which is much more modular and easy to change or reuse in the future.

Hope it makes sense.